pexpect/pxssh.py

```diff
@@ -358,7 +358,43 @@ def login (self, server, username, password='', terminal_type='ansi',
                         if spawn_local_ssh==False:
                             tunnel = quote(str(tunnel))
                         ssh_options = ssh_options + ' -' + cmd_type + ' ' + str(tunnel)
-        cmd += " %s -l %s %s" % (ssh_options, username, server)
```

Comment:
This line needs to be added back, the rebase removed it.

Reply:
I misread that and removed it in the merge from master into patch-1 because cmd is overwritten on 396 (I think). The custom ssh command option looks like it needs to be put in at 396 if I'm not mistaken.